plugins/rest.js

```diff
@@ -307,7 +313,13 @@ module.exports = function(finish, o) {
 			queryAllowed: { // Fields and their allowed contents (post remap)
 				'$limit': {number: true},
 				'$populate': {scalar: true, scalarCSV: true, array: true},
-				'$select': {scalar: true, scalarCSV: true, array: true},
+				'$select': {scalar: true, scalarCSV: true, array: true,
+					// Mapping "1" to boolean see issue #14
+					format: f => {
+						if (!_.isObject(f)) return f;
+						return _(f).mapValues(v => (v == 'true' || v == '1')).value();
```

Comment:
Could maybe just convert them if they equal those values, if not return whatever they happen to be.